extensions/wikia/Discussions/maintenance/dumpForumData.php

```diff
@@ -45,40 +47,19 @@ public function dumpPages() {
 		$pages = $this->dumper->getPages();
 
 		foreach ( $pages as $id => $data ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$id" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)